lib/feed.xml

```diff
@@ -53,26 +53,12 @@
       <content type="html" xml:base="{{ post.url | prepend: url_base | xml_escape }}">{{ post.content | strip | xml_escape }}</content>
 
       {% assign post_author = post.author | default: post.authors[0] | default: site.author %}
-      {% assign post_author_email = false %}
-      {% assign post_author_uri = false %}
+      {% assign post_author = site.data.authors[post_author] | default: post_author %}
```

Comment:
> Maybe if it is just a string, and not a reference, we assign it to author name.

I don't know of a way in Liquid to see what type a variable is.

> it may make sense to have a shared `jekyll-author` gem

Totally agree, but as @GarthDB mentioned, let's get this bug fix out first.